packages/cli/lib/index.js

```diff
@@ -28,41 +27,100 @@ process.on('unhandledRejection', err => {
 	error(err.stack || err.message);
 });
 
-let prog = sade('preact').version(pkg.version);
+const prog = sade('preact').version(pkg.version);
 
-const buildCommand = prog
+prog
 	.command('build [src]')
 	.describe(
 		'Create a production build. You can disable "default: true" flags by prefixing them with --no-<option>'
-	);
-commands.buildOptions.forEach(option => {
-	buildCommand.option(option.name, option.description, option.default);
-});
-buildCommand.action(commands.build);
+	)
+	.option('--src', 'Specify source directory', 'src')
```

Comment:
I think I remember it originally being done this way, but I don't know why it was changed to the version it currently is

Probably older than the new `sade` option

Reply:
Indeed, we had the commands in this format but I think `sade`'s option was just missed, so validation was done the way it was. It's a pretty long ReadMe without a table of contents, easy enough to do.

Alternatively, maybe there was an idea for better type checking on args down the road? `sade`/`mri` are a bit "loose" with coercing input types, leading to a few odd situations. Just guessing though; if we wanted to do that there's probably simpler ways to validate the schema than this setup.